src/Sema.zig

```diff
@@ -7350,6 +7352,11 @@ fn funcCommon(
             });
         }
 
+        // TODO do this check in AstGen if https://github.com/ziglang/zig/issues/11834 is accepted
```

Comment:
This check will still need to happen in `Sema` even with #11834 since the callconv expression can still be any comptime expression, just not one affected by comptime parameters.

Reply:
so you could for example have it be a global or function call, as long as it didnt use the parameters?

Reply:
Yes, you can look at `kernel32.zig` for an example of where this is a useful feature.